assets/cohere.yaml

```diff
@@ -145,7 +145,26 @@
       support at cohere.ai.
 
 - type: model
-  name: Cohere Representation model
+  name: Command
+  # General
+  organization: Cohere
+  description: >
+    This model is a generative model optimized to follow commands in the prompt.
+  dependencies: [Cohere Base]
```

Comment:
This is great, but you will need all the fields (and you can leave them empty if needed) as in the Cohere base entry. I.e. all of those fields are mandatory to define a model entry.

Reply:
also add modality and so on.